source/sync/configure/pause-or-terminate-sync.txt

```diff
@@ -155,12 +155,15 @@ devices.
 
    .. step:: Confirm That You Want to Terminate Sync
 
-      In the modal that appears, type "Terminate Sync" in the text entry, 
-      and then click the  :guilabel:`Terminate Sync` button.
+      In the modal that appears, type "Terminate Sync" in the text
+      entry, and then click the  :guilabel:`Terminate Sync` button.
 
       .. figure:: /images/terminate-sync-modal.png
        :alt: the terminate sync model
 
+      If you are using :ref:`UI deployment drafts <deployment-draft>`
+      then you must deploy the draft before any changes take effect.
```

Comment:
It's probably worth being explicit here since we're talking about terminating sync, instead of general "before any changes take effect."

```suggestion
      then you must deploy the draft to actually terminate Sync.
```

Reply:
Ah yeah I was more explicit before but then it felt weird to say terminate sync twice in a row. Oh well I'm sure our readers will understand :)